packages/frontend/package.json

```diff
@@ -58,8 +58,10 @@
     "test": "jest",
     "lint": "eslint --ext .js --ext .jsx .",
     "fix": "eslint --ext .js --ext .jsx . --fix",
+    "format": "prettier --write",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "precommit": "lint-staged"
```

Comment:
Any reason not to run unit tests here as well?

Reply:
I'm of the opinion that doing this might make development slightly frustrating as these tests will run for every commit and prevent committing if a test fails. Sometimes, you may be aware that a test is broken and may plan to fix it in another commit. Perhaps instead of using the `pre-commit` hook, we can use `pre-push`?  So it only runs when you try to push code to remote.

Reply:
Thought about this again. On second thought, this can be used to run staged tests within a commit. Is this what you meant?

Reply:
`pre-push` and `pre-commit` both sound good to me 👍 . I think we should run all tests though to catch any dependencies on the changed code. Maybe until speed becomes a bottleneck.